database/buildSchema/44_data_view.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE data_view (
     row_restrictions jsonb DEFAULT '{}',
     table_view_include_columns varchar[],
     table_view_exclude_columns varchar[],
+    table_search_columns varchar[],
```

Comment:
`table_search_columns` specifies which columns the "Search" component should include. (Must be simple "text" columns)